app/controllers/spree/credit_cards_controller.rb

```diff
@@ -26,6 +26,8 @@ def update
       else
         update_failed
       end
+      rescue ArgumentError => e
```

Comment:
We need to watch out here. Since this is a very broad exception class we will rescue many cases we don't want to and that will be annoying, like probably errors in specs. What's the problem that we're trying to solve?